packages/components/src/components/checkbox/checkbox.stories.ts

```diff
@@ -1,16 +1,19 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 const { argTypes, parameters } = storybookDefaults('sd-checkbox');
 const { generateTemplate } = storybookTemplate('sd-checkbox');
 const { overrideArgs } = storybookHelpers('sd-checkbox');
 
+/**
+ * Allows users to toggle an option on or off.
```

Comment:
Cannot find this description in Figma. Please align with Design.

In Figma we have "Allows users to select one or more values from a set of options. A standalone checkbox is used to turn a setting or option on or off."

Reply:
The agreed description is the one in code which as also already been adopted in design.

Reply:
Now we have: "Used to toggle an option on or off." in Figma 🤣

Reply:
Updated